src/components/SearchResultsCategory.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+
+export default function SearchResultsCategory(): JSX.Element {
+  const classes = useStyles();
+  return (
+    <div className={classes.paper}>
```

Comment:
You might also be able to just use the Paper component. I haven't been able to figure out why some of the Material-UI examples use Paper and others use div with a 'paper' style.

Reply:
Interesting, I had assumed Paper was a content container like Card, and have seen Paper mostly used in the Grid examples 
https://material-ui.com/components/grid/#grid
and that classes.paper was similar to css classes.